README.md

```diff
@@ -9,3 +9,152 @@
+
+## Installation
+
+Install the latest release from [Homebrew](https://brew.sh/), [Chocolatey](https://chocolatey.org/packages/gardenctl) or [GitHub Releases](https://github.com/gardener/gardenctl-v2/releases).
+
+### Install using Package Managers
+
+```sh
+# Homebrew (macOS and Linux)
+brew install gardener/tap/gardenctl-v2
+
+# Chocolatey (Windows)
+choco install gardenctl-v2
+```
+
+Attention `brew` users: `gardenctl-v2` uses the same binary name as the legacy `gardenctl` (`gardener/gardenctl`) CLI. If you have an existing installation you should remove it with `brew uninstall gardenctl` before attempting to install `gardenctl-v2`. Alternatively, you can choose to link the binary using a different name. If you try to install without removing or relinking the old installation, brew will run into an error and provide instructions how to resolve it.
+
+### Install from Github Release
+
+If you install via GitHub releases, you need to put the `gardenctl` binary on your path. The other install methods do this for you.
+
+```bash
+# Example for macOS
+
+# set operating system and architecture
+os=darwin # choose between darwin, linux, windows
+arch=amd64 # choose between amd64, arm64
+
+# Get latest version. Alternatively set your desired version
+version=$(curl -s https://raw.githubusercontent.com/gardener/gardenctl-v2/master/LATEST)
+
+# Download gardenctl
+curl -LO https://github.com/gardener/gardenctl-v2/releases/download/$(curl -s https://raw.githubusercontent.com/gardener/gardenctl-v2/master/LATEST)/"gardenctl_v2_${os}_${arch}"
+
+# Make the gardenctl binary executable
+chmod +x "./gardenctl_v2_${os}_${arch}"
+
+# Move the binary in to your PATH
+sudo mv "./gardenctl_v2_${os}_${arch}" /usr/local/bin/gardenctl
+```
+
+## Configure Gardenctl
+
+`gardenctl` requires a configuration file. The default location is in `~/.garden/gardenctl-v2.yaml`.
+
+You can modify this file directly using the `gardenctl config` command. It allows adding, modifying and deleting gardens.
+
+Example `config` command:
+``` bash
+# Adapt the path to your kubeconfig file for the garden cluster
+export KUBECONFIG=~/path/to/garden-cluster/kubeconfig.yaml
+
+# Fetch cluster-identity of garden cluster
+CLUSTER_IDENTITY=$(kubectl -n kube-system get configmap cluster-identity -ojsonpath={.data.cluster-identity})
+
+# Configure garden cluster
+gardenctl config set-garden $CLUSTER_IDENTITY --kubeconfig $KUBECONFIG
+```
+This command will create or update a garden with the provided identity and kubeconfig path of your garden cluster.
+
+### Example Config
+
+```yaml
+gardens:
+- identity: landscape-dev # Unique identity of the garden cluster. See cluster-identity ConfigMap in kube-system namespace of the garden cluster
+  kubeconfig: ~/path/to/garden-cluster/kubeconfig.yaml
```

Comment:
I would remove the `~`

Reply:
no particular reason, it's just an example. We also can get rid of it.. we should then also adapt https://github.com/gardener/gardenlogin/blob/master/README.md accordingly.
But it should be supported though

Reply:
Let's keep it in for now. It shows that we have support for the shorthand path to the user's home directory.

Reply:
Hmm. But then I would write `~/relative/path/to/kubeconfig.yaml`

Reply:
And I would remove `garden-cluster` because this is completely clear from the context.